src/components/pages/form-intermediate-step-page/FormIntermediateStepPage.tsx

```diff
@@ -0,0 +1,169 @@
+        }
+    };
+
+    const currentStepData = buildStepObjectFromNestedStructure(currentPage);
+
+    const themedPageHeaderProps = {
+        ...props,
+        data: {
+            ...props.data,
+            taxonomy: [],
+            customCategory: getTranslations('application'),
+        },
+    };
+
+    return (
+        <div className={styles.formIntermediateStepPage}>
+            <ThemedPageHeader
+                contentProps={themedPageHeaderProps}
+                showTimeStamp={false}
+            />
+            <div className={styles.content}>
+                <div className={styles.stepOptionsWrapper}>
+                    <ParsedHtml htmlProps={currentStepData.editorial} />
+                    {currentStepData.stepsHeadline && (
+                        <Heading level="3" size="small" spacing>
+                            {currentStepData.stepsHeadline}
+                        </Heading>
+                    )}
+                    <ul className={styles.stepList}>
+                        {currentStepData.steps.map((step, index) => {
+                            const isLinkExternal = !!step.externalUrl;
+                            const onClick = isLinkExternal
+                                ? null
+                                : (e) => nextHandler(e, index, step.label);
+
+                            const href = isLinkExternal
+                                ? step.externalUrl
+                                : '#';
+
+                            return (
+                                <li key={step.label}>
+                                    <LinkPanel
+                                        href={href}
+                                        onClick={onClick}
+                                        className={styles.stepOption}
+                                    >
+                                        <LinkPanel.Title>
+                                            {step.label}
+                                        </LinkPanel.Title>
+                                        <LinkPanel.Description>
+                                            {step.explanation}
+                                        </LinkPanel.Description>
+                                    </LinkPanel>
+                                </li>
+                            );
+                        })}
+                    </ul>
+                </div>
+                <div className={styles.buttonGroup}>
+                    {currentPage > 0 && (
+                        <Button
+                            onClick={(e) => backHandler(e)}
+                            variant="tertiary"
+                            as="a"
+                            href="#"
+                            className={styles.backButton}
+                        >
+                            {getTranslations('back')}
+                        </Button>
+                    )}
+                </div>
+            </div>
+        </div>
+    );
+};
```

Comment:
Navigasjonen mellom stegene her er litt problematisk for brukeropplevelsen, ettersom back/forward funksjonalitet i browser ikke vil fungere som forventet. Tror dette bør løses på en annen måte.